testapp/helpers.go

```diff
@@ -128,7 +128,7 @@ func (a *App) StateDoesNotContain(t *testing.T, height uint64, kvs map[string]st
 		}).Marshal()
 		require.NoError(t, err)
 		resp, err := a.Query(context.Background(), &abcitypes.RequestQuery{
-			Path:   QueryPath, // TODO is there a way to find this programmatically?
+			Path:   QueryPath,
```

Comment:
I looked into generating and passing through the query path programmatically, however since this use case is only for this helper file I decided it wasn't worth the overhead to refactor.

For visibility, there were two potential approaches that I looked into:
1. Use the `ServiceName` and "Get" `MethodName` from [`_QueryService_serviceDesc`](https://github.com/polymerdao/monomer/pull/130/files#diff-77a41b8c9fc4875149ea5fc72fb2487b4d02031b00df1d3de313d3934cb1418bR211-R222) to dynamically build this path. However, the method name field is numerically indexed so our path would break if "Get" isn't the first method name.
2. [Create](https://github.com/polymerdao/monomer/pull/130/files#diff-77a41b8c9fc4875149ea5fc72fb2487b4d02031b00df1d3de313d3934cb1418bR163) a new query service client and then [call](https://github.com/polymerdao/monomer/pull/130/files#diff-77a41b8c9fc4875149ea5fc72fb2487b4d02031b00df1d3de313d3934cb1418bR167) the `Get` method to return the get response. However, this would require a good bit of additional boilerplate code to create the `grpc1.ClientConn` used to build the query service client.

Reply:
Hmm interesting. Thanks for investigating! This will be valuable information when we add more APIs to the Cosmos SDK app in the `integrations` package.